node/src/test/scala/com/wavesplatform/transaction/smart/script/ScriptCompilerV1Test.scala

```diff
@@ -26,8 +26,8 @@ class ScriptCompilerV1Test extends PropSpec with EitherValues with Inside {
   }
 
   property("fails on unsupported version") {
-    val script = scriptWithVersion("8".some)
-    ScriptCompiler(script, isAssetScript = false, estimator) shouldBe Left("Illegal directive value 8 for key STDLIB_VERSION")
+    val script = scriptWithVersion("9".some)
```

Comment:
`StdLibVersion.VersionDic.max.id + 1` is more versatile choice

Reply:
Fixed